spring-boot/admin/src/main/resources/templates/admin/dashboard.html

```diff
@@ -92,6 +92,7 @@
                     loadStatisticsPerApiConsumerInDescendingOrderByNoOfCalls(dateRangeUrlSuffix);
                     loadChart('/admin/dashboard/httpResponseStatsChart' + dateRangeUrlSuffix, 'httpStatsChartCanvas');
                     loadChart('/admin/dashboard/userStatsDoughnutChart' + dateRangeUrlSuffix, 'userStatsChartCanvas');
+                    loadChart('/admin/dashboard/userStatsDoughnutChart' + dateRangeUrlSuffix, 'callsPerServiceChartCanvas');
```

Comment:
@ChristophBaierATB please change this to `loadChart('/admin/dashboard/serviceStatsChartCanvas ' + dateRangeUrlSuffix, 'callsPerServiceChartCanvas');`